modules/ROOT/pages/dw-mule-functions-p.adoc

```diff
@@ -14,8 +14,7 @@ Environment variables.
 
 The `p` function returns a `null` value if the property is not set or if the function does not find the property.
 
-For more on this topic, see
-https://docs.mulesoft.com/mule-runtime/4.2/configuring-properties[Configure Properties].
+For more on this topic, see xref:configuring-properties.adoc[Configure Properties].
```

Comment:
Note that we need to use the HTTP URLs for DW function docs because the source is used in other places besides our docs.

Reply:
And we have to update the DW function docs in the data-weave repo. Maybe we could use the "latest" URL instead of 4.2?

Reply:
fixed in 0f7291a